mkfs/dump.c

```diff
@@ -145,7 +145,7 @@ int main(int argc, char **argv)
                       SECTOR_SIZE,
                       infinity,
                       h,
-                      closure(h, bread, fd, get_fs_offset(fd)),
+                      sg_wrapped_block_reader(closure(h, bread, fd, get_fs_offset(fd)), SECTOR_OFFSET, h),
                       closure(h, bwrite, fd),
```

Comment:
This needs a call to `init_sg(h)`, otherwise sg_heap will be NULL and sg_wrapped_block_reader() will crash when trying to allocate a closure.

Reply:
Fixed